incidenter/program.py

```diff
@@ -0,0 +1,145 @@
+from os import getenv
+import re
+
+from autokitteh import next_event, subscribe
+from autokitteh.slack import slack_client
+
+from height import add_task_message
+from height import create_task
+from zoom import create_meeting
+
+
+CHANNEL_PREFIX = getenv("CHANNEL_PREFIX")
+
+slack_client = slack_client("slack")
+
+pattern = re.compile(r"<.+> incident *(.*)")
```

Comment:
1. Start at the beginning of the string
2. Don't be fooled by `">"` later in the string
3. `"\s*"` is more clear than `" *"`
```suggestion
pattern = re.compile(r"^<.+?> incident\s*(.*)")
```

Reply:
done